README.md

```diff
@@ -4,13 +4,18 @@ VPF stands for Video Processing Framework. It’s set of C++ libraries and Pytho
 
 VPF also supports exporting GPU memory objects such as decoded video frames to PyTorch tensors without Host to Device copies. Check the [Wiki page](https://github.com/NVIDIA/VideoProcessingFramework/wiki/Building-from-source) on how to build from source.
 
-## Installation
-
-VPF works on Windows and Linux. The requirements are as follows
-
-- CUDA Toolkit (npp)
-- [FFMPEG](https://github.com/FFmpeg/FFmpeg/) (with libavfilter>=7.110.100)
-- C++ compiler
+## Prerequisites
+VPF works on Linux(Ubuntu 20.04 and Ubuntu 22.04 only) and Windows
+
+- NVIDIA display driver: 525.xx.xx or above
+- CUDA Toolkit 11.2 or above 
+  - CUDA toolkit has driver bundled with it e.g. CUDA Toolkit 12.0 has driver `530.xx.xx`. During installation of CUDA toolkit you could choose to install or skip installation of the bundled driver. Please choose the appropriate option.
+- FFMPEG
+  - [Compile FFMPEG with shared libraries](https://docs.nvidia.com/video-technologies/video-codec-sdk/12.0/ffmpeg-with-nvidia-gpu/index.html) or download pre-compiled binaries from a source you trust. During VPF’s “pip install”(mentioned in sections below) you need to provide a path to the directory where FFMPEG got installed.
```

Comment:
Shouldn't the system package option be mentioned? I feel that people might not understand that `libav*` is FFmpeg.

a source you trust (e.g. `apt install  libavfilter-dev libavformat-dev libavcodec-dev libswresample-dev libavutil-dev` on Ubuntu)

Reply:
yes, it should be